docs/pages/10_view.adoc

```diff
@@ -0,0 +1,629 @@
+
+=== View action
+
+A view action is a visible component within a user interface that represents a specific operation or task that can be triggered by user interaction through a clickable button. The view action defines a piece of code, currently implemented in Java, that executes when the button is clicked. The full list of modifiers can be found in the xref:appx_syntax.adoc#view-action[Appendix].
+
+*Syntax:*
+
+[subs="quotes"]
+----
+[purple]#**action**# &#x5B;&lt;output>|[purple]#**void**#] &lt;name> [purple]##**(**##&#x5B;&lt;parameter>] &#x5B;[purple]##**choices:**##&lt;choices>][purple]##**)**##
+    &#x5B;[purple]##**throws**## &lt;error1> &#x5B;[purple]##**,**##&lt;error2>] ...]
+
+    &#x5B;[purple]##**static:**##([purple]##**true**##|[purple]##**false**##)]
+    &#x5B;[purple]##**enabled:**##&lt;enabled>]
+    &#x5B;[purple]##**update:**##([purple]##**true**##|[purple]##**false**##)]
+    &#x5B;[purple]##**delete:**##([purple]##**true**##|[purple]##**false**##)]
+
+    &#x5B;[purple]##**label:**##&lt;label>]
+    &#x5B;[purple]##**icon:**##&lt;icon>]
+    &#x5B;[purple]##**hidden:**##&lt;hidden>]
+    &#x5B;[purple]##**width:**##&lt;width>]
+    &#x5B;[purple]##**redirect:**##&lt;redirect>][purple]##**;**##
+----
+
+The basic structure of a view action declaration follows the same pattern as xref:07_transfer.adoc#action[Transfer actions]. Both view and transfer actions can include modifiers such as `static`, `enabled`, `update`, and `delete` to define their behavior and characteristics. These modifiers control aspects such as whether the action is not associated with a specific instance (`static`) or its availability for execution (`enabled`).
+
+In both cases, developers start with the `action` keyword followed by the return type of the action (<output>), which can be a view type, a union, or `void` if the action does not return any value. Then, the name of the action (<name>) is specified, followed by an optional input parameter (<parameter>) enclosed in parentheses. This parameter represent the data that the action requires for its execution.
+
+Other modifiers such as `label`, `icon`, `hidden` and `width` are specific to view actions, allowing developers to customize the appearance of the actions in the user interface. See more about them in <<Common visual modifiers>> chapter.
+
+The `redirect` modifier in the context of a view action specifies the name of an access where the returned view is available. When the action is invoked, and its execution is completed, the user is redirected to the specified access, enabling seamless navigation within the application. This modifier allows developers to define the target destination for the user after the action has been executed successfully. By specifying the access name, developers can ensure that users are directed to the appropriate view or page based on the outcome of the action.
```

Comment:
Is redirect optional, or mandatory? If it is optional what happens to mapped return type views? Actions break relation chains.

Reply:
fixed:
"If there is no `redirect` modifier specified, the returned value is read only. Thus, it cannot be changed and actions defined on it cannot be invoked."